### ISSUE TITLE
Errors of import map from Roadrunner containing traffic lights?

### ISSUE THREAD

**Issue by huhaocong1** [carla-simulator/carla#5257]
Hi, I'm having some issues importing map generated from Roadrunner. 

The environment of my computer is :
Ubuntu-20.04
UE-4.26.2
Calra-0.9.13
RoadRunner2021b

I created one test map,contains lot of props of RoadRunner.After i imported the map (use "make import")to carla,in the edit mode,i could not seen any traffic_light or signal_post(in RoadRunner i putted all kinds of traffic lights and signal posts)???But i could seen other props i putted in the map.When i "play" the map,i saw two traffic lights(only two,and it was hand-up by a signal-post),why??

What should i do?

Hopes for your help!!

Thanks!

**Comment by RocketDK** [comment]
Similar problem on my side with Carla 0.9.13. No traffic signs are shown in CARLA after importing a map from RoadRunner. I also tried to create a minimal map with only a small road and a traffic sign, and still no luck. Also: `make import` doesn't raise any errors.

**Comment by huhaocong** [comment]
这个问题一直没有解决，不知道是不是carla官方与UE4之间的“协商”问题？
我后来直接使用RoadRunner Plugin方式导入地图（为了使用RoadRunner里的那些便利的地图功能而妥协了）！！
或者，只在RoadRunner内编辑路面功能（包括道路、便道、地面），然后其他所有的编辑工作（绿植、建筑、标牌等）直接转移到carla内部制作！

**Comment by xiaoyang666** [comment]
> 

这个问题也困扰了我很久，用插件还是不能解决这个问题

**Comment by heesungsung** [comment]
Did you solve the problem?
I have the same issue.  Traffic lights and poles do not appear in the Carla.

**Comment by jannine92** [comment]
Are there any updates from anyone? I'm facing the same issue with RoadRunner R2022a and Carla 0.9.13 and I need to import the maps by using the `make import` way.

**Comment by i14d4** [comment]
> Similar problem on my side with Carla 0.9.13. No traffic signs are shown in CARLA after importing a map from RoadRunner. I also tried to create a minimal map with only a small road and a traffic sign, and still no luck. Also: `make import` doesn't raise any errors.

I have the same issue as already described. Does anyone has a solution for how a map from roadrunner can be import including traffic lights?
kind regards and thanks a lot in advance !

**Comment by goodgoodliving** [comment]
Yes, I've also encountered this problem. Similar to street lights, these poles don't appear on the map
My versions are Roadrunner 2022b and carla 0.9.13 .and I use the **make import** way.

**Comment by goodgoodliving** [comment]
> 这个问题一直没有解决，不知道是不是carla官方与UE4之间的“协商”问题？ 我后来直接使用RoadRunner Plugin方式导入地图（为了使用RoadRunner里的那些便利的地图功能而妥协了）！！ 或者，只在RoadRunner内编辑路面功能（包括道路、便道、地面），然后其他所有的编辑工作（绿植、建筑、标牌等）直接转移到carla内部制作！

我想问一下这些编辑工作如何在carla内部制作啊？？？

**Comment by goodgoodliving** [comment]
@huhaocong 我想问一下这些编辑工作如何在carla内部制作啊？？？

**Comment by goodgoodliving** [comment]
> 这个问题一直没有解决，不知道是不是carla官方与UE4之间的“协商”问题？ 我后来直接使用RoadRunner Plugin方式导入地图（为了使用RoadRunner里的那些便利的地图功能而妥协了）！！ 或者，只在RoadRunner内编辑路面功能（包括道路、便道、地面），然后其他所有的编辑工作（绿植、建筑、标牌等）直接转移到carla内部制作！

@huhaocong 我想问一下这些编辑工作如何在carla内部制作啊？？？

**Comment by goodgoodliving** [comment]
> Hi, I'm having some issues importing map generated from Roadrunner.
> 
> The environment of my computer is : Ubuntu-20.04 UE-4.26.2 Calra-0.9.13 RoadRunner2021b
> 
> I created one test map,contains lot of props of RoadRunner.After i imported the map (use "make import")to carla,in the edit mode,i could not seen any traffic_light or signal_post(in RoadRunner i putted all kinds of traffic lights and signal posts)???But i could seen other props i putted in the map.When i "play" the map,i saw two traffic lights(only two,and it was hand-up by a signal-post),why??
> 
> What should i do?
> 
> Hopes for your help!!
> 
> Thanks!

@huhaocong1 请问你解决了吗？？

**Comment by huhaocong1** [comment]
打开ue4，找到地图，编辑，拖拽




------------------&nbsp;原始邮件&nbsp;------------------
发件人:                                                                                                                        "carla-simulator/carla"                                                                                    ***@***.***&gt;;
发送时间:&nbsp;2023年3月27日(星期一) 下午3:53
***@***.***&gt;;
***@***.******@***.***&gt;;
主题:&nbsp;Re: [carla-simulator/carla] Errors of import map from Roadrunner containing traffic lights? (Issue #5257)





  
这个问题一直没有解决，不知道是不是carla官方与UE4之间的“协商”问题？ 我后来直接使用RoadRunner Plugin方式导入地图（为了使用RoadRunner里的那些便利的地图功能而妥协了）！！ 或者，只在RoadRunner内编辑路面功能（包括道路、便道、地面），然后其他所有的编辑工作（绿植、建筑、标牌等）直接转移到carla内部制作！
  
@huhaocong 我想问一下这些编辑工作如何在carla内部制作啊？？？

**Comment by goodgoodliving** [comment]
> 打开ue4，找到地图，编辑，拖拽
> […](#)
> ------------------&nbsp;原始邮件&nbsp;------------------ 发件人: "carla-simulator/carla" ***@***.***&gt;; 发送时间:&nbsp;2023年3月27日(星期一) 下午3:53 ***@***.***&gt;; ***@***.******@***.***&gt;; 主题:&nbsp;Re: [carla-simulator/carla] Errors of import map from Roadrunner containing traffic lights? (Issue #5257) 这个问题一直没有解决，不知道是不是carla官方与UE4之间的“协商”问题？ 我后来直接使用RoadRunner Plugin方式导入地图（为了使用RoadRunner里的那些便利的地图功能而妥协了）！！ 或者，只在RoadRunner内编辑路面功能（包括道路、便道、地面），然后其他所有的编辑工作（绿植、建筑、标牌等）直接转移到carla内部制作！ @huhaocong 我想问一下这些编辑工作如何在carla内部制作啊？？？ — Reply to this email directly, view it on GitHub, or unsubscribe. You are receiving this because you authored the thread.Message ID: ***@***.***&gt;

@huhaocong1    好的感谢！！！     roadrunner导入的方法还是没有成功是吧？

**Comment by huhaocong1** [comment]
不知道啊，我早就离职了，不做那个了




------------------&nbsp;原始邮件&nbsp;------------------
发件人:                                                                                                                        "carla-simulator/carla"                                                                                    ***@***.***&gt;;
发送时间:&nbsp;2023年3月27日(星期一) 下午4:43
***@***.***&gt;;
***@***.******@***.***&gt;;
主题:&nbsp;Re: [carla-simulator/carla] Errors of import map from Roadrunner containing traffic lights? (Issue #5257)





  
打开ue4，找到地图，编辑，拖拽
 …
 ------------------&nbsp;原始邮件&nbsp;------------------ 发件人: "carla-simulator/carla" @.&gt;; 发送时间:&nbsp;2023年3月27日(星期一) 下午3:53 @.&gt;; @.@.&gt;; 主题:&nbsp;Re: [carla-simulator/carla] Errors of import map from Roadrunner containing traffic lights? (Issue #5257) 这个问题一直没有解决，不知道是不是carla官方与UE4之间的“协商”问题？ 我后来直接使用RoadRunner Plugin方式导入地图（为了使用RoadRunner里的那些便利的地图功能而妥协了）！！ 或者，只在RoadRunner内编辑路面功能（包括道路、便道、地面），然后其他所有的编辑工作（绿植、建筑、标牌等）直接转移到carla内部制作！ @huhaocong 我想问一下这些编辑工作如何在carla内部制作啊？？？ — Reply to this email directly, view it on GitHub, or unsubscribe. You are receiving this because you authored the thread.Message ID: @.***&gt;
  
@huhaocong1    好的感谢！！！     roadrunner导入的方法还是没有成功是吧？

**Comment by goodgoodliving** [comment]
> Did you solve the problem? I have the same issue. Traffic lights and poles do not appear in the Carla.

@heesungsung  Did you solve the problem?

**Comment by goodgoodliving** [comment]
> Are there any updates from anyone? I'm facing the same issue with RoadRunner R2022a and Carla 0.9.13 and I need to import the maps by using the `make import` way.

 @jannine92  Did you solve the problem?

**Comment by jannine92** [comment]
No, I still have the same problem.

**Comment by sainttelant** [comment]
any updated for this issue?
